ts-core/src/Differential.ts

```diff
@@ -134,6 +134,7 @@ const functionRegistry: { [key: string]: ServiceRegistryFunction } = {};
 
 class PollingAgent {
   private errorCount = 0;
+  private idleCycleCount = 0;
```

Comment:
Hey, should we make this a `idleTimeout`? 

Since the polling interval can be configured, it might be confusing to think in terms of cycles.

Was there a specific reason you thought of doing cycles instead of time?

Reply:
>Was there a specific reason you thought of doing cycles instead of time?

Nope, no reason will update to a timeout 👍